src/test/steps/checkCountOfItems.js

```diff
@@ -22,6 +27,65 @@ When(
   }
 );
 
+When(
+  /^Send request to the search\/'(.*)'\/count endpoint without conditional$/,
```

Comment:
better to add a type of request to step name
```suggestion
  /^Send GET request to the search\/'(.*)'\/count endpoint without conditional$/,
```

Reply:
Yes, I agree with this remark. All methods, unless explicitly specified, are GET methods. The ones on GET are marked accordingly.
when fixing the project globally, I will add the GET method to all requests.